api/src/services/script-runner.service.ts

```diff
@@ -1105,6 +1105,86 @@ export class ScriptRunnerService {
     return { success: true };
   }
 
+  /**
+   *
+   * @param req incoming request object
+   * @returns successDTO
+   * @description add duplicates information to lottery email
+   */
+  async addDuplicatesInformationToLotteryEmail(
+    req: ExpressRequest,
+  ): Promise<SuccessDTO> {
+    const requestingUser = mapTo(User, req['user']);
+    await this.markScriptAsRunStart(
+      'add duplicates information to lottery email',
+      requestingUser,
+    );
+
+    const updateTranslation = async (
```

Comment:
lets not stack helper functions in a function. This should be pulled out into a stand alone helper function